godtools/Managers/LanguagesManager.swift

```diff
@@ -0,0 +1,79 @@
+//
+//  LanguagesManager.swift
+//  godtools
+//
+//  Created by Ryan Carlson on 4/18/17.
+//  Copyright © 2017 Cru. All rights reserved.
+//
+
+import Foundation
+import UIKit
+import Alamofire
+import PromiseKit
+import Spine
+import MagicalRecord
+
+class LanguagesManager: NSObject {
+    static let shared = LanguagesManager()
+    
+    let path = "/languages"
+    let serializer = Serializer()
+    
+    var languages = [Language]()
+    
+    override init() {
+        super.init()
+        serializer.registerResource(LanguageResource.self)
+    }
+    
+    func loadFromDisk() -> Promise<[Language]> {
+        languages = Language.mr_findAll() as! [Language]
```

Comment:
Yes. As I wrote the second manager class, I realized the code was similar in many ways to the first.

I don't like to prematurely abstract though because it can actually create more refactoring work in the long run. I typically like to extract super classes after 3rd or sometimes 2nd implementation because by that point it's easier to see the patterns in the architecture, and it finds that good point of saving time in the future w/o creating unnecessary refactoring.

So all of that to say, yes good idea.. but I'm not going to do it just quite yet :)